### ISSUE TITLE
Allow MustLoadMessageFile to accept an optional langTag

### ISSUE THREAD

**Issue by jtruong2** [nicksnyder/go-i18n#347]
`MustLoadMessageFile()` takes a path that follows the file naming convention of `xxx.en.json` and automatically parses the language tag. This incorrectly retrieves the language tag if the path does not follow the convention. In my case, the locales path is `locales/en/translation.json`. Passing this path to `MustLoadMessageFile()` will result in the langTag being `translation` instead of `en`. It'd be super helpful if either the function could also parse the aforementioned path or accept a langTag to override the automatic parsing.

**Comment by nicksnyder** [comment]
Yeah, I wish I had not as tightly coupled the locale detection to the file name. I'm not going to make backward incompatible changes to existing method signatures, so the only option would be to add a new method.

The best workaround is what I mentioned here: https://github.com/nicksnyder/go-i18n/issues/376#issuecomment-3703185137.
